pkg/cri/resource-manager/policy/builtin/topology-aware/pod-preferences.go

```diff
@@ -59,6 +62,8 @@ func podIsolationPreference(pod cache.Pod, container cache.Container) (bool, boo
 }
 
 // podSharedCPUPreference checks if a container wants to opt-out from exclusive allocation.
+// The first return value indicates the container is ok to share resources (true).
+// FIXME(rojkov): what is elevate?
```

Comment:
I think an accurate description should be: The first return value indicates if the container prefers to opt-out from exclusive (sliced-off or isolated) CPU allocation even if it was otherwise eligible for it.

The second return value, elevate, indicates how much to elevate the actual allocation of the container in the tree of pools. IOW, how many levles to go up in the tree starting at the best fitting pool, before assigning the container to an actual pool. That preference is currently ignored.

Reply:
Updated.